common/defaults.go

```diff
@@ -1353,7 +1353,7 @@ func (r *RetryPolicyConfig) SetDefaults(defaults *RetryPolicyConfig) error {
 		if defaults != nil && defaults.Delay != 0 {
 			r.Delay = defaults.Delay
 		} else {
-			r.Delay = Duration(100 * time.Millisecond)
+			r.Delay = Duration(0 * time.Millisecond)
```

Comment:
I think this default is shared between upstream and network failsafe policies. is it safe to set it as `0` for upstream as well?